docs/source/configuration.rst

```diff
@@ -976,6 +976,13 @@ show_only_online_users
 If set to ``true``, only online users will be shown in the contacts roster.
 Users with any other status (e.g. away, busy etc.) will not be shown.
 
+show_send_button
+----------------------
+
+* Default:  ``false``
+
+If set to ``true``, a send button will be visible to activate send message operation via mouse, without keyboard.
```

Comment:
`to activate send message operation` sounds super technical.

Please change to:
```
If set to ``true``, a button will be visible which can be clicked to send a message.
```